tests/requirements.txt

```diff
@@ -0,0 +1,27 @@
+cookiecutter==0.9.0
+coverage==3.7.1
+ddt==0.8.1
+diff-cover==0.7.2
+Django==1.4.16
+django_nose==1.2
+fs==0.5.0
+lazy==1.2
+mock==1.0.1
+nose==1.3.4
+nose-parameterized==0.3.5
+pep8==1.5.7
+pylint==0.28
+pypng==0.0.17
+rednose==0.4.1
+requests==2.4.3
+selenium==2.44.0
+simplejson==3.6.5
+unicodecsv==0.9.4
+webob==1.4
```

Comment:
@e-kolpakov Right, most of these are transitive dependecies. The only way to get a consistent, repeatable environment is to freeze all direct *and* transitive dependencies. Take `webob` for example. `xblock-sdk` declares that it depends on `webob`, but it doesn't declare the version. If I remove `webob` from `tests/requirements.txt` and `pip install` it today, I will get `webob 1.4`, the most recent version. But if version `2.0` gets released next week and you `pip install` into a fresh virtual environment next week, you'll get `2.0`. Your tests will fail and you'll blame me for committing a broken test suite.

Until we decide to update one of the direct dependencies of this project, say `xblock-sdk`, we won't run into any conflicts. When we do want to bump one of the existing dependencies, the person that bumps it needs to update the frozen dependencies as well. The easiest way to do that is to install direct dependencies into a fresh virtual environment and then use `pip freeze` to produce a new `test/requirements.txt` file. Unfortunately python's tooling doesn't make this as straightforward as ruby's bundler or node's npm.